### DISCUSSION TITLE
Blowfish not compatible with the latest Hugo version

### DISCUSSION THREAD

**Discussion by natashahrycan** [nunocoracao/blowfish#1635] [Q&A] [answered]
I recently installed in Windows Hugo (v0.129.0) and Blowfish, turns out Blowfish is compatible to Hugo only up to v0.128.2. Has anyone had experience with downgrading Hugo from Powershell? It would be preferable for Blowfish to upgrade for compatibility, but in the worst case I will downgrade Hugo.

**Comment by nunocoracao** [comment, marked as answer]
@natashahrycan don't worry too much about it. It's just a warning for users to know that something might be broken. Next release will include an update to it. I have that system in place to make sure we all know about any breaking changes everytime there's a new Hugo release so that they can be fixed at theme level.

**Reply by natashahrycan** [reply]
That's lovely to know, thanks for the answer!

**Comment by RonnyG2121** [comment]
Hello.
I having this problem too. My Hugo version is v0.145.0-666444f0a52132f9fec9f71cf25b441cc6a4f355+extended windows/amd64

**Reply by Emp5r0R** [reply]
Same, I am too experiencing the same warning for this version

**Reply by rroethof** [reply]
Same issue, running github actions with hugo v0.145.0-666444f0a52132f9fec9f71cf25b441cc6a4f355 linux/amd64 BuildDate=2025-02-26T15:41:25Z VendorInfo=gohugoio
Not cool, because my automated deploy actions are simply failing now

**Reply by Emp5r0R** [reply]
> Same issue, running github actions with hugo v0.145.0-666444f0a52132f9fec9f71cf25b441cc6a4f355 linux/amd64 BuildDate=2025-02-26T15:41:25Z VendorInfo=gohugoio Not cool, because my automated deploy actions are simply failing now

Recently, mine are too

**Reply by CollieIsCute** [reply]
same issue too, upgrade 0.145 to 0.146 cause this error msg

**Reply by CollieIsCute** [reply]
I think #2055 can solve this issue

**Comment by rroethof** [comment]
I had to change my CI to use older version of Hugo:

      - name: Setup Hugo
        uses: peaceiris/actions-hugo@v3
        with:
          hugo-version: '0.145.0' #latest is broken
          extended: true

**Reply by Z4kr-Sec** [reply]
Where am I supposed to  make that change, I have hugo version 146 and I'm getting the error
